colossalai/shardformer/policies/bert.py

```diff
@@ -142,7 +142,7 @@ def unembedding() -> List:
                 weight="decoder.weight",
                 bias="decoder.bias",
                 replace_layer=col_nn.Linear1D_Col,
-        # gather_output=True,
+                gather_output=True,
```

Comment:
make this configurable from the user api.

Reply:
https://github.com/hpcaitech/ColossalAI/blob/5153e181d749adec09d7455ae26ce3a962607c90/colossalai/shardformer/shard/sharder.py#L153

https://github.com/hpcaitech/ColossalAI/blob/5153e181d749adec09d7455ae26ce3a962607c90/colossalai/shardformer/shard/shard_config.py#L18

gather_output in policy means this layer can be gatherd, gather_output in shardconfig mean whether to gatherd